seal.go

```diff
@@ -80,16 +111,21 @@ func SealKeyToTPM(tpm *tpm2.TPMContext, dest string, mode SealMode, key []byte)
 		pinFlags = existing.PinFlags
 	}
 
-	// Convert policy inputs in to individual event digests
-	//  TODO
-
-	// Use event digests, the event log and GRUB data to generate PCR digests
-	//  TODO: Use policy inputs rather than the current PCR values
-	//  TODO: Generate digests for other PCRs
-	_, digests, err := tpm.PCRRead(tpm2.PCRSelectionList{
-		tpm2.PCRSelection{Hash: defaultHashAlgorithm, Select: tpm2.PCRSelectionData{7}}})
-	if err != nil {
-		return fmt.Errorf("cannot read PCR values: %v", err)
+	var secureBootDigests tpm2.DigestList
+	if policy != nil {
+		var err error
```

Comment:
(nitpick^2) you could move `var err error` out of this if next to var secureBootDigests and remove this and the other var err error below.

Reply:
Ok, I've fixed this one.